### PR TITLE
[css-backgrounds] Initial values and inheritance

### PR DESCRIPTION
**Pull request by ewilligers**
Test that CSS Backgrounds and Borders properties have the initial
values given in the spec. None of the properties inherit.

https://drafts.csswg.org/css-backgrounds/

Identified bugs:

repeated keywords in initial value
~https://bugzilla.mozilla.org/show_bug.cgi?id=1501261~

border-image initial values
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19381805/

box-shadow serialization order
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19381950/

border-image-outset initial value
https://bugs.chromium.org/p/chromium/issues/detail?id=898203

### CODE REVIEW

**Comment by ewilligers** [comment]
@fantasai  Please review

**Comment by ewilligers** [comment]
@dbaron Please review

File: css/css-backgrounds/inheritance.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+<div id="container">
+  <div id="target"></div>
+</div>
+<style>
+  #reference {
+    column-rule-style: dotted; /* Avoid column-rule-width computed style 0 */
+    column-rule-width: medium;
+  }
+  #container {
+    border-style: solid;
```
Comment (dbaron) [inline css/css-backgrounds/inheritance.html]: please add a comment here and 3 lines below about `border-width` style... which I assume is what this is about... or whatever it is about?

File: css/css-backgrounds/inheritance.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+<link rel="help" href="https://drafts.csswg.org/css-backgrounds-3/#property-index">
+<meta name="assert" content="Properties inherit or not according to the spec.">
+<meta name="assert" content="Properties have initial values according to the spec.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/inheritance-testcommon.js"></script>
+</head>
+<body>
+<div id="reference"></div>
+<div id="container">
```
Comment (dbaron) [inline css/css-backgrounds/inheritance.html]: perhaps comment that container and target are for the use of the functions in inheritance-testcommon.js?

File: css/css-backgrounds/inheritance.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  #container {
+    border-style: solid;
+  }
+  #target {
+    border-style: solid;
+  }
+</style>
+<script>
+const transparentColor = 'rgba(0, 0, 0, 0)'; // https://www.w3.org/TR/css-color-3/#transparent
+const mediumWidth = getComputedStyle(reference).columnRuleWidth; // e.g. 3px
```
Comment (dbaron) [inline css/css-backgrounds/inheritance.html]: please call `getElementById("reference")` rather than depending on the global scope polluter

**Review by dbaron — APPROVED** [review]

**Pull request merged**